client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -34,6 +35,7 @@ function getChecklistCompletionProgress( tasks: Task[] ) {
 const Sidebar = ( { siteSlug }: { siteSlug: string | null } ) => {
 	let siteName = '';
 	let topLevelDomain = '';
+	const site = useSite();
```

Comment:
We can also move this `useSite()` import directly into the checklist.tsx component instead of passing as prop.

Reply:
@agrullon95 - Good call. I've moved this.